core/benchmarks/ZStandardBenchmark-results.txt

```diff
@@ -2,48 +2,48 @@
-Decompression 10000 times from level 1 without buffer pool            635            635           0          0.0       63471.5       1.0X
-Decompression 10000 times from level 2 without buffer pool            637            638           1          0.0       63693.2       1.0X
-Decompression 10000 times from level 3 without buffer pool            637            638           1          0.0       63687.9       1.0X
-Decompression 10000 times from level 1 with buffer pool               545            545           0          0.0       54463.9       1.2X
-Decompression 10000 times from level 2 with buffer pool               544            545           1          0.0       54405.3       1.2X
-Decompression 10000 times from level 3 with buffer pool               544            545           1          0.0       54399.6       1.2X
+Decompression 10000 times from level 1 without buffer pool            595            595           0          0.0       59493.7       1.0X
+Decompression 10000 times from level 2 without buffer pool            596            596           1          0.0       59575.1       1.0X
+Decompression 10000 times from level 3 without buffer pool            596            597           1          0.0       59607.5       1.0X
+Decompression 10000 times from level 1 with buffer pool               541            542           1          0.0       54117.6       1.1X
+Decompression 10000 times from level 2 with buffer pool               541            542           1          0.0       54088.7       1.1X
+Decompression 10000 times from level 3 with buffer pool               541            542           0          0.0       54106.0       1.1X
 
-OpenJDK 64-Bit Server VM 17.0.14+7-LTS on Linux 6.8.0-1020-azure
+OpenJDK 64-Bit Server VM 17.0.14+7-LTS on Linux 6.8.0-1021-azure
 AMD EPYC 7763 64-Core Processor
 Parallel Compression at level 3:          Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Parallel Compression with 0 workers                  61             62           0          0.0      475889.8       1.0X
-Parallel Compression with 1 workers                  43             45           2          0.0      338238.7       1.4X
-Parallel Compression with 2 workers                  42             45           2          0.0      327766.3       1.5X
-Parallel Compression with 4 workers                  43             45           1          0.0      334407.0       1.4X
-Parallel Compression with 8 workers                  44             47           1          0.0      345047.0       1.4X
-Parallel Compression with 16 workers                 50             52           1          0.0      387573.7       1.2X
+Parallel Compression with 0 workers                  78             78           0          0.0      607618.7       1.0X
```

Comment:
cc @yaooqinn Will the results of this test also fluctuate?

Reply:
Results seem reasonable

Reply:
Got it ~